bin/compile

```diff
@@ -173,6 +192,8 @@ summarize_build() {
   if $NODE_VERBOSE; then
     list_dependencies "$BUILD_DIR"
   fi
+
+  mmeasure 'modules.size' "$(measure_size)"
```

Comment:
K. FWIW: I checked for that before commenting but found places where "" is used w/o interpolation. My search was also not exhaustive. ;-)